README.md

```diff
@@ -134,7 +134,7 @@ Lastly, cases where you wish to call `trackEvent` with only the eventName parame
     Klaviyo.sharedInstance.setUpUserEmail(userEmail: "john.smith@example.com")
 ```
 ## Sending Push Notifications
-To be able to send push notifications, you must add two snippets of code to your application. One to register users for push notifications, and one that will send Klaviyo their tokens. 
+To be able to send push notifications, you must add few snippets of code to your application. One to register users for push notifications, and one that will send Klaviyo their tokens. 
```

Comment:
```suggestion
To be able to send push notifications, you must add a few snippets of code to your application. One to register users for push notifications, one that will send Klaviyo their tokens, and one to something something...
```

Reply:
```suggestion
To be able to send push notifications, you must add a few snippets of code to your application. One to register users for push notifications, one that will send resulting push tokens to Klaviyo, and one final optional snippet so that notifications show up when the app is foregrounded.
```